pages/common/cloc.md

```diff
@@ -14,3 +14,7 @@
 - Compare 2 directory structures and count the differences between them:
 
 `cloc --diff {{/directory/one}} {{/directory/two}}`
+
+- Ignore files that are ignored by VCS, such as git:
```

Comment:
```suggestion
- Ignore files that are ignored by VCS, such as .git/ directory:
```

Reply:
Thanks for your response! How about this?

    Ignore files that are ignored by VCS, such as files specified in ".gitignore":

That's closer to what I meant to say.

Reply:
BTW, I signed the CLA.

Reply:
Remove the quote marks and it'll be fine.

Reply:
Done. Please take a look.

Reply:
@zdroid, I am going to resolve this review due @jduan change it as it was requested. Excuse me the intereference.